aws/s3_test.go

```diff
@@ -410,16 +418,18 @@ func TestNukeS3Bucket(t *testing.T) {
 					objectCount:       10,
 					objectBatchsize:   1000,
 					shouldNuke:        true,
+					shouldError:       false,
 				},
 			},
 			{
 				bucketType + "_BatchObjects_ValidBatchSize",
 				TestNukeS3BucketArgs{
 					isVersioned:       isVersioned,
 					checkDeleteMarker: false,
-					objectCount:       10,
+					objectCount:       30,
 					objectBatchsize:   5,
```

Comment:
Bumped the object count here to make sure the delete in page routine works even when there are many pages (in this case at least 6).